frontend/js/utils/battle-detail.js

```diff
@@ -0,0 +1,15 @@
+const showTeams = (battle, user) => {
```

Comment:
What are you trying to achieve here? Is there another way to do this? I noticed this exactly equal to the one Deborah wrote

Reply:
Just to ensure that the trainers can't access each other's team, and can only see each other's team after the battle is over. I did it with her, she was helping me, that's why it's the same

Reply:
Even though you are both working together, the code should never be the same. I would like to suggest for you refactor this having [this in mind](https://www.vinta.com.br/blog/2018/flat-success-path/).